src/Common/src/Interop/Windows/kernel32/Interop.CreateFile.cs

```diff
@@ -14,7 +14,7 @@ internal partial class Kernel32
         /// <summary>
         /// WARNING: The private methods do not implicitly handle long paths. Use CreateFile.
         /// </summary>
-        [DllImport(Libraries.Kernel32, EntryPoint = "CreateFileW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
+        [DllImport(Libraries.Kernel32, EntryPoint = "CreateFileW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false,ExactSpelling = true)]
```

Comment:
The meaning of Exact Spelling in AOT is a bit different , we kind of re purposed it. But in general case it use EntryPoint as it is when ExactSpelling is specifed.